conf/mirrors.yaml

```diff
@@ -167,10 +167,6 @@
   cat: a
   name: Nanyang Technological University, Singapore
   url: http://jambu.spms.ntu.edu.sg/sage/
-- active: true
-  cat: e
-  name: GARR, Italy
-  url: http://sage.mirror.garr.it/mirrors/sage/
 - active: false
```

Comment:
I just checked: despite `active: false` this mirror works and is up to date: http://sage.mirror.garr.it/mirrors/sage/

Why remove? Actually, it should be set to to `active: true`

Reply:
The GARR mirror has `active: true` (line 170); line 174 is about the next mirror.
Also, the "onlinedirect" mirror (in Sofia, Bulgaria), which was at L173 when the
issue was opened, is now further down since other mirrors (like the GARR one)
were added in the meanwhile. Note that #139 sets the "onlinedirect" mirror in
Sofia to `active: false`, fixing #16.